Services/UI/OJS.Services.Ui.Business/Implementations/ParticipantsBusinessService.cs

```diff
@@ -34,6 +35,18 @@ public class ParticipantsBusinessService : IParticipantsBusinessService
         this.datesService = datesService;
     }
 
+    public Task<ContestParticipantsViewModel> GetPracticeParticipantsCount(int contestId)
+        => Task.FromResult(new ContestParticipantsViewModel
```

Comment:
It's not a good practice to return ViewModel in a service, but the bigger issue here is that you return the model populated with only one of the values. Everyone that calls this method should have to use just this value (as the other will be missing) and the model becomes useless.
```
public class ContestParticipantsViewModel
{
    public int CompeteParticipantsCount { get; set; }

    public int PracticeParticipantsCount { get; set; }
}
```
I would return just an integer, as the name of the method suggests and use the value where I need it to populate my view model.

Reply:
Done.